mysql_scripts/table_creation/FullDB.sql

```diff
@@ -129,32 +134,32 @@ CREATE TABLE PasswordReset (
 	FOREIGN KEY (userId) REFERENCES User(userId)
 );
 
-CREATE TABLE Privilege (
-	privilegeId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
+CREATE TABLE Permission (
```

Comment:
I didn't like the term privilege either. It doesn't make as much sense as permission in my opinion.